.travis.yml

```diff
@@ -66,7 +66,6 @@ before_script:
  - cd ..
 
 script:
- - cd /home/travis/build/RicinApp/Ricin
  - make autogen
  - make install
```

Comment:
Please run this as `sudo` since it needs to copy files in `/usr/bin` & `/usr/share/` ;)